### PR TITLE
Build certs search results page 

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Remove redirect
- Build search results page for `/certifications`
- Nice to have - when selecting the filters the list automatically updates. This isn't on the current site so would be an additional feature maybe looked at once the other pages are built. 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/certification
- Use the drop down and search bar to search for hardware. 
- Try selecting different filters
- Check all links work correctly

## Issue / Card

Fixes [#146](https://github.com/canonical-web-and-design/certification.ubuntu.com/issues/146)

### CODE REVIEW

File: webapp/certification/views.py [hunk 1]
```diff
@@ -124,21 +124,21 @@ def certification_home():
...
 
         # Populate filter numbers
-        for model in enumerate(results):
-            form_filters[model["category"]] += 1
-            release_filters[model["release"]] += 1
-            vendor_filters[model["make"]] += 1
+        for index, model in enumerate(results):
+            form_filters[model["category"]] = int(index + 1)
+            release_filters[model["release"]] = int(index + 1)
+            vendor_filters[model["make"]] = int(index + 1)
```
Comment (carkod) [inline webapp/certification/views.py:141]: Can you revert this? and simply replace 
`for index, model in enumerate(results):`
`for model in results:`
Thanks

**Comment by anasereijo** [comment]
@bethcollins92 this is looking great! 2 tiny things:

- [ ] can you left-align the pagination as per my design?
- [ ] make 'SHOW', 'CERTIFIED FOR UBUNTU' and 'VENDOR' the same font colour as the search results title (eg '21 – 40 OF 58 RESULTS')

**Comment by bethcollins92** [comment]
Thanks @anasereijo, made those changes 👍🏼

**Comment by anasereijo** [comment]
LGTM 👍  I will give a +1 but things will probably change after UX review, so I will review it again later on

**Review by carkod — APPROVED** [review]
LGTM

**Pull request merged**